test/fixtures/rest/app/apis/users/posts/answer/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+exports.index = function* () {
```

Comment:
@popomore Updated the readme, now it is clearly for what I have done. The `apis/users/posts/replies/answer.js` is not supported at the moment. The maximum nesting layers are three with is `users`, `posts`, `replies`. `answer` was never supported. 

I do have a testing use case to test on 404 for `answer.js` though, please, checkout `rest.test.js`. Hopefully it makes senses to you.